docs/manual/docs/cmd/login.md

```diff
@@ -13,7 +13,7 @@ login [options]
 Option|Description
 ------|-----------
 `--help`|output usage information
-`-t, --authType [authType]`|The type of authentication to use. Allowed values `certificate,deviceCode,password`. Default `deviceCode`
+`-t, --authType [authType]`|The type of authentication to use. Allowed values `certificate,deviceCode,password,identity`. Default `deviceCode`
```

Comment:
IMO, the type of `managed` is a better choice. All authentication methods result in an identity.

Reply:
Since we also have `deviceCode` rather than just `device` or `code`, what if we used `managedIdentity` for clarity?

Reply:
@waldekmastykarz . @garrytrinder , @appieschot if you agree, I can easily change it.

Reply:
For clarity, we settled on `identity` initially as this was then consistent with how it is implemented in the Azure CLI.

https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest#sign-in-with-a-managed-identity

That said happy, to change to `managedIdentity`.

Reply:
Good point @garrytrinder. Totally missed that. If we can align with az cli then it would make sense to keep things as similar as possible to avoid unnecessarily introducing new terms.

Reply:
Ugh! Sometimes Microsoft cannot get out of its own way...

Reply:
We leave it to identity. Will resolve this.